cnr.sh

```diff
@@ -1,72 +1,85 @@
 #!/bin/bash
+#set -e
 
-function update_cnr {
-    echo "wip: install/update cnr-cli"
+LATEST="v0.4.0"
 
+function download_or_noop {
+  if [ ! -e "$HELM_PLUGIN_DIR/cnr" ]; then
```

Comment:
I think it's really confusing for someone to run `helm plug install github.com/app-registry/helm-plugin` and then immediately get prompted the first time they try to use it that they didn't actually install it.